shared/src/main/java/com/couchbase/lite/LogDomain.java

```diff
@@ -17,9 +17,51 @@
 //
 package com.couchbase.lite;
 
+import java.util.EnumSet;
+
 /**
- * Log domain. The log domains here are tentative and subject to change.
+ * Log domain
  */
 public enum LogDomain {
```

Comment:
Please revert LogDomain Enum to the original as there is to do bitwise at the Enum level. EnumSet has already done the job.

```Java
public enum LogDomain {
    ALL, DATABASE, QUERY, REPLICATOR, NETWORK
}
```